### PR TITLE
add  aihuishou.jpeg

### PR DESCRIPTION
**Pull request by qijianbo010**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by qijianbo010** [comment]
@googlebot I signed it!

**Comment by ericvn** [comment]
@qijianbo010 You also need to add a reference in content/en/about/community/customers/index.md. It should be added in place alphabetically.

**Comment by qijianbo010** [comment]
change content/en/about/community/customers/index.md

File: content/en/about/community/customers/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ skip_seealso: true
 Here are just a few of the organizations we know are using Istio. If you want to edit this page, [please send a pull request!](https://github.com/istio/istio.io/tree/master/content/en/about/community/customers)
 
 <div class="logo-gallery">
+    {{< company_logo link="https://www.aihuishu.com" logo="./aihuishou.jpeg" alt="Aihuishou" >}}
```
Comment (ericvn) [inline content/en/about/community/customers/index.md:13]: Link returns not found for me.Suspect it's missing a `o` and should be https://www.aihuishou.com.

Comment (qijianbo010) [reply content/en/about/community/customers/index.md]: Oh my God

**Comment by qijianbo010** [comment]
what is tide？

**Comment by ericvn** [comment]
Tide will run after your PR is approved. If all other required items have a green check, then your PR will merge after approval.

**Comment by ericvn** [comment]
@istio/wg-docs-maintainers-chinese Thoughts on this? Since the link is to a website I cannot read, I want to make sure this is appropriate.

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Review by frankbu — APPROVED** [review]

**Pull request merged**